features/finders.feature

```diff
@@ -288,12 +288,19 @@ Feature: Filtering documents
     And the page has results region
 
   @javascript
-  Scenario: Facets should be hidden by default on finders except all content
+  Scenario: Facets should not be hidden by default on finders except all content
     When I view the research and statistics finder
     And I should not see a "Show more search options" link
     Then I view the all content finder
     And I should see a "Show more search options" link
 
+  @javascript
+  Scenario: Facets should expand when clicking on the "Show more search options" link on all content
+    When I view the all content finder
+    And I should see a "Show more search options" link
+    Then I click to expand all facets
+    And I should see a "Show fewer search options" link
```

Comment:
While we're at it, is it worth checking to see that after the link is clicked, the content we expect is visible?

Reply:
That would create a dependency on the facets that we display for the finder. It's not a very big thing and I'm happy to go either way on this.

Reply:
That's a good point. Is it possible to check that any one of several likely facets is visible? So the test would pass if a checkbox OR a select box OR a text input is suddenly visible in the space? You could even do a generic 'if there's any kind of input in this space' check?

Reply:
I've met you halfway in my latest commit 🙂